README.md

```diff
@@ -1,3 +1,102 @@
-# rukpak
+# RukPak
 
-Rukpak runs in a Kubernetes cluster and defines an API for installing cloud native bundle content 
+RukPak runs in a Kubernetes cluster and defines an API for installing cloud native bundle content. 
+
+## Introduction 
+
+RukPak is a pluggable solution for the packaging and distribution of cloud-native content and supports advanced strategies for installation, updates, and policy.
+The project provides a content ecosystem for installing a variety of artifacts, such as Git repositories, Helm charts, OLM bundles, and more onto a Kubernetes cluster. 
+These artifacts can then be managed, scaled, and upgraded in a safe way to enable powerful cluster extensions. 
+
+At its core, RukPak is a small set of APIs, packaged as Kubernetes CustomResourceDefinitions, and controllers that watch for those APIs. These APIs express what 
+content is being installed on-cluster and how to create a running instance of the content. 
+
+## Components 
+
+RukPak is composed of a few primary APIs: `Bundle`, `Instance`, and `Provisioner/ProvisionerClass`
```

Comment:
I'd remove the mention of Provisioner here - it seems to imply that it's a concrete type vs. the controller responsible for watching these primary APIs.

Reply:
Might be worth linking the API type (`./api`) definition directory here.

Reply:
We can ignore linking the API directory for now - but I'd like to remove the mention of `Provisioner/ProvisionerClass` and just have `ProvisionerClass` documented here before merging this.